__tests__/__snapshots__/ordCdsrc.test.js.snap

```diff
@@ -4,55 +4,6 @@ exports[`Tests for default ORD document when .cdsrc.json is present Successfully
 {
   "$schema": "https://sap.github.io/open-resource-discovery/spec-v1/interfaces/Document.schema.json",
   "apiResources": [
-    {
```

Comment:
@zongqichen : Since the `visibility` of AdminService in the __tests__\bookshop\srv\admin-service.cds is set now to `private`, none of it's resources(apiResources, evenResources) will be present in generated ORD Document.
The rule is: `private` and `internal` resources are hidden, only the `public` ones will be shown in ORD Doc.

Reply:
As suggested, AdminService is now **public**, CatalogService is **private** and CinemaService is **internal**.